From acb39c429e804bcf3c6a478a3850185364fe9abf Mon Sep 17 00:00:00 2001
From: Atul Dhudase <quic_adhudase@quicinc.com>
Date: Tue, 16 Jul 2024 15:45:44 +0530
Subject: [PATCH] QCLINUX: arm64: dts: qcom: Add board-id and msm-id for
 QCM6490 boards

Add board-id and msm-id for QCM6490-idp and rb3gen2 for now. This is only a
workaround, that shall be replaced by the compatible string
check approach to pick the correct DTB.

Upstream-Status: Pending
---
 arch/arm64/boot/dts/qcom/qcm6490-idp.dts     | 4 ++++
 arch/arm64/boot/dts/qcom/qcs6490-rb3gen2.dts | 4 ++++
 2 files changed, 8 insertions(+)

diff --git a/arch/arm64/boot/dts/qcom/qcm6490-idp.dts b/arch/arm64/boot/dts/qcom/qcm6490-idp.dts
index f154b9a401fc..78a6030b2f67 100644
--- a/arch/arm64/boot/dts/qcom/qcm6490-idp.dts
+++ b/arch/arm64/boot/dts/qcom/qcm6490-idp.dts
@@ -33,6 +33,10 @@ / {
 	compatible = "qcom,qcm6490-idp", "qcom,qcm6490";
 	chassis-type = "embedded";
 
+	/* This will be deprecated soon */
+	qcom,msm-id = <497 0x10000>, <498 0x10000>, <475 0x10000>, <515 0x10000>;
+	qcom,board-id = <34 0>, <34 0x1>, <34 0x2>, <34 0x600>, <34 0x601>,  <34 0x602>;
+
 	aliases {
 		serial0 = &uart5;
 	};
diff --git a/arch/arm64/boot/dts/qcom/qcs6490-rb3gen2.dts b/arch/arm64/boot/dts/qcom/qcs6490-rb3gen2.dts
index bb3671ce2515..09348b0ffabf 100644
--- a/arch/arm64/boot/dts/qcom/qcs6490-rb3gen2.dts
+++ b/arch/arm64/boot/dts/qcom/qcs6490-rb3gen2.dts
@@ -30,6 +30,10 @@ / {
 	compatible = "qcom,qcs6490-rb3gen2", "qcom,qcm6490";
 	chassis-type = "embedded";
 
+	/* This will be deprecated soon */
+	qcom,msm-id = <497 0x10000>, <498 0x10000>, <475 0x10000>, <515 0x10000>;
+	qcom,board-id = <32 0x5>, <32 0x605>;
+
 	aliases {
 		serial0 = &uart5;
 	};
-- 
2.25.1

